src/icrar/leap-accelerate/ms/MeasurementSet.h

```diff
@@ -51,10 +53,12 @@ namespace icrar
         std::unique_ptr<casacore::MSColumns> m_msc;
         std::unique_ptr<casacore::MSMainColumns> m_msmc;
 
+        int m_stations;
+
     public:
-        MeasurementSet(std::string filepath);
-        MeasurementSet(const casacore::MeasurementSet& ms);
-        MeasurementSet(std::istream& stream);
+        MeasurementSet(std::string filepath, boost::optional<int> overrideNStations);
+        MeasurementSet(const casacore::MeasurementSet& ms, boost::optional<int> overrideNStations);
+        MeasurementSet(std::istream& stream, boost::optional<int> overrideNStations);
```

Comment:
These `optional`s could be defaulted to `boost::none` or similar to make "simple" construction easier to use.

Reply:
I had this defaulted before but then realized all of my measurement sets need this overriden and shouldn't be overlooked. I'll double check whether the repaired measurement set I received yesterday also needs to be overriden.

Reply:
updated where I needed to override so don't mind either way now